src/setupTest.js

```diff
@@ -9,13 +9,16 @@ import { configure as configureI18n } from '@edx/frontend-platform/i18n';
 import { configure as configureLogging, MockLoggingService } from '@edx/frontend-platform/logging';
 import { getConfig, mergeConfig } from '@edx/frontend-platform';
 import { configure as configureAuth, MockAuthService } from '@edx/frontend-platform/auth';
+import envConfig from '../env.config';
 import messages from './i18n';
 
 jest.mock('@edx/frontend-platform/react/hooks', () => ({
   ...jest.requireActual('@edx/frontend-platform/react/hooks'),
   useTrackColorSchemeChoice: jest.fn(),
 }));
 
+mergeConfig(envConfig);
```

Comment:
Jest doesn't rely on webpack to merge the JS-based config into ConfigDocument, so we have to implement it here. I might create a separate file that does this merge in the Jest directory if it makes more sense.

Reply:
I think this is reasonable to have in `setupTest.js`. I guess one question/suggestion might be whether it could be in the base `setupTest.js` upstream in `@edx/frontend-build` so that it'll be applicable to all consuming MFEs. I imagine other consuming MFEs of `env.config` would otherwise need to follow this same approach? Is there an opportunity to make it part of your patch and upstream contribution?

Reply:
From my understanding, `frontend-platform` is able to consume `frontend-build`, but `frontend-build` wouldn't be able to use `frontend-platform`'s `mergeConfig` out of the box because Build isn't set up to `import` modules. Not to say it isn't a good idea, but I believe we'd need to convert everything in Build to use `import` instead of `require` so we can bring `frontend-platform`'s `getConfig` in, right?

Reply:
Yeah, good call out. I agree we can't do much here with `setupTest` here after all. Perhaps the action item then is possibly about documentation in your upstream `@edx/frontend-build` contribution and/or in `@edx/frontend-platform` to ensure consumers know to include this in their `setupTest.js` files if the intent is to have those settings come through in tests.